modules/battenberg/1.1/src/reference_correction.py

```diff
@@ -0,0 +1,38 @@
+##### ATTRIBUTION #####
```

Comment:
this file is not present in version 1.1 and needs to be removed

Reply:
done